Procfile

```
@@ -0,0 +1,2 @@
+rails: rails s -p 3000
```

`-b 0.0.0.0` をつけてください。
https://qiita.com/Masato338/items/f162394fbc37fc490dfb

```suggestion
rails: bundle exec rails s -b 0.0.0.0 -p 3000
```

コマンドはこっちで
```
docker-compose run --rm --service-ports web bundle exec foreman start -f Procfile
```